opba-embedded-starter/src/deb/control/control

```diff
@@ -5,4 +5,4 @@ Priority: low
 Architecture: all
 Description: Provides Access to Banking API
 Maintainer: Francis Pouatcha <Francis.Pouatcha@adorsys.com>
-Depends: default-jre | java6-runtime
\ No newline at end of file
+Depends: openjdk-21-jre
```

Comment:
To require Java 21 or any higher version, use: 
`Depends: java-runtime (>= 21)`